cpp/src/Slice/Parser.cpp

```diff
@@ -770,23 +756,29 @@ namespace
 }
 
 CommentPtr
-Slice::Contained::parseComment(bool stripMarkup) const
+Slice::Contained::parseComment(function<string(string)> linkFormatter, bool includeDeprecatedMetadata, bool stripMarkup)
+    const
 {
-    return parseComment(_comment, stripMarkup);
-}
+    CommentPtr comment = make_shared<Comment>();
 
-CommentPtr
-Slice::Contained::parseComment(const string& text, bool stripMarkup) const
-{
-    if (text.empty())
+    // Check for deprecated metadata and add it to the comment if necessary.
```

Comment:
Please don't do that. I removed this behavior in #2970, and looking at this PR, it appears I forgot do removed it from Swift and MATLAB. Please removed there too.

Reply:
In fairness, the comment code for Swift and MATLAB was completely separate from all the other comment code, so it's not surprising.

I removed all the includeDeprecated stuff, and am happy to see it go!